### ISSUE TITLE
training acc is 93%, but verification acc is 65%?

### ISSUE THREAD

**Issue by clscy** [deepinsight/insightface#478]
I try to train my own dataset, it comes such problem.
![screenshot from 2018-12-18 09-14-05](https://user-images.githubusercontent.com/22907560/50126119-8aeb1680-02a6-11e9-9a66-ecf1e6e4deba.png)
Anybody knows why?

**Comment by clscy** [comment]
thank you for your answer. the calidation set has some types and distribution with training set. I think that the phenomenon maybe category imbalance problem.some one category contain 80 samples, but some one category contain 20 samples. the softmax loss may need to replace to focal loss.

**Comment by TccccD** [comment]
@clscy hello, After you have changed the distribution of the validation set data, is the obtained acc normal?

**Comment by clscy** [comment]
after that, the validation acc is 0.85%, the focal loss can improve the acc higher than softmax.

**Comment by nanhui69** [comment]
> after that, the validation acc is 0.85%, the focal loss can improve the acc higher than softmax.

what‘s dataset do you test?
